pkg/execution/queue/item.go

```diff
@@ -63,6 +63,10 @@ type Item struct {
 	// Payload stores item-specific data for use when processing the item.  For example,
 	// this may contain the function's edge for running a step.
 	Payload any `json:"payload,omitempty"`
+
+	// Metadata is used for storing additional metadata related to the queue item.
+	// e.g. tracing data
+	Metadata map[string]string
```

Comment:
Does this mean that all trace attributes must be strings?

Reply:
Yes, because the `MapCarrier` type for span context and baggage are always `map[string]string`

https://pkg.go.dev/go.opentelemetry.io/otel@v1.16.0/propagation#MapCarrier